pom.xml

```diff
@@ -395,7 +395,7 @@
             <Embed-Directory>lib</Embed-Directory>
             <!--Only @Plugin classes in the export packages will be included as plugin-->
             <_exportcontents>
-              io.cdap.plugin.*;
+              io.cdap.plugin.servicenow.source.*,io.cdap.plugin.servicenow.connector.*;
```

Comment:
I would expect doing this would cause problems where the sink plugin isn't visible. This can instead be `io.cdap.plugin.servicenow.*`

Reply:
`io.cdap.plugin.servicenow.*`
This makes the ServiceNow Sink Plugin visible in the studio. 

The sink plugin was't ready for release so we skipped it from the export contents.

> I would expect doing this would cause problems where the sink plugin isn't visible. 

Could you please elaborate on some of the problems?

Reply:
Removed the `@Plugin` annotation and added a comment there about the Sink plugin not being ready yet. 9248f60ddf04f053ff92e673bcf3148d73d2d8ef